pkgs/by-name/ce/certdump/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildDotnetModule,
+  fetchFromGitHub,
+  stdenv,
+}:
+
+buildDotnetModule rec {
+  pname = "certdump";
+  version = "unstable-2023-12-25";
+
+  src = fetchFromGitHub {
+    owner = "secana";
+    repo = "CertDump";
+    rev = "a834da24ee18503109631d836540a2b0cb481517";
+    sha256 = "sha256-86s6KLP0DliKOr0fVId7SGN333b7HkiL5p/q0vazwMc=";
```

```suggestion
    hash = "sha256-86s6KLP0DliKOr0fVId7SGN333b7HkiL5p/q0vazwMc=";
```
https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub